src/pages/profile/profile.js

```diff
@@ -20,15 +26,22 @@ function Profile() {
         }
     }
     async function onAddUser() {
+        if (firstName.length < 1 && lastName.length < 1 && email.length < 1) {
+            setError({
+                firstName: "please fill in your first name.",
+                lastName: "please fill in your last name.",
+                email: "please fill in your email.",
+            });
+            console.log("issue with the inputs", firstName, firstName.length);
+            console.log(error);
```

Comment:
`console.error(error)` will make the error show up as an error statement instead of an info statement in the console.